.github/release/release-main-config.json

```diff
@@ -0,0 +1,60 @@
+{
+  "packages": {
+    ".": {
+      "package-name": "zenstack-monorepo",
+      "component": "Monorepo"
+    },
+    "packages/ide/jetbrains": {
+      "package-name": "jetbrains",
+      "component": "JetBrains_IDE"
+    },
+    "packages/language": {
+      "package-name": "@zenstackhq/language",
+      "component": "Language"
+    },
+    "packages/misc/redwood": {
+      "package-name": "@zenstackhq/redwood",
+      "component": "Redwood"
+    },
+    "packages/plugins/openapi": {
+      "package-name": "@zenstackhq/openapi",
+      "component": "OpenAPI_Plugin"
+    },
+    "packages/plugins/swr": {
+      "package-name": "@zenstackhq/swr",
+      "component": "SWR_Plugin"
+    },
+    "packages/plugins/tanstack-query": {
+      "package-name": "@zenstackhq/tanstack-query",
+      "component": "Tanstack_Query_Plugin"
+    },
+    "packages/plugins/trpc": {
+      "package-name": "@zenstackhq/trpc",
+      "component": "tRPC_Plugin"
+    },
+    "packages/runtime": {
+      "package-name": "@zenstackhq/runtime",
+      "component": "Runtime"
+    },
+    "packages/sdk": {
+      "package-name": "@zenstackhq/sdk",
+      "component": "SDK"
+    },
+    "packages/server": {
+      "package-name": "@zenstackhq/server",
+      "component": "Server"
+    },
+    "packages/testtools": {
+      "package-name": "@zenstackhq/testtools",
+      "component": "Test_Tools"
+    }
+  },
+  "pull-request-footer": "This PR was generated by [Release-Please](https://github.com/googleapis/release-please), and approved by the ZenStack Team.",
+  "bump-minor-pre-major": true,
+  "bump-patch-for-minor-pre-major": true,
+  "sequential-calls": true,
+  "separate-pull-requests": true,
```

Comment:
Often a feature/fix in ZenStack involves changes in multiple packages. Maybe using a combined PR makes more sense in this case?

Reply:
So "seperate-pull-requests" creates a pull request for each of your projects on any release, as show in the forked repo. While it does add a small bit of extra work to merge in changes which effect multiple packages, it does allow you to release individual packages at different times or rates. 

If you have them all as one PR, it will instead track the changes and version them in a single PR, tagging all of them and making release notes at once when you release. 

It's totally personal preference, I personally would rather spend an extra couple of minutes merging multiple PR's vs releasing all my packages at once :)

Reply:
Got it. I'll keep it as is and tweak it in the future.